### PR TITLE
Support ReadOnlySequence as input

### PR DESCRIPTION
**Pull request by gfoidl**
Fixes https://github.com/gfoidl/Base64/issues/61

### CODE REVIEW

**Comment by gfoidl** [comment]
@ycrumeyrolle I gave the `ReadOnlySequence`-thing a shot, please have a look.

**Comment by gfoidl** [comment]
A preview package is here -> https://www.myget.org/feed/gfoidl/package/nuget/gfoidl.Base64/1.1.0-preview-1

File: source/gfoidl.Base64/ThrowHelper.cs [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 using System.Buffers;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Resources;
 
+#if NETSTANDARD2_0
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Method, Inherited = false)]
+    public sealed class DoesNotReturnAttribute : Attribute { }
```
Comment (ycrumeyrolle) [inline source/gfoidl.Base64/ThrowHelper.cs]:
```suggestion
    internal sealed class DoesNotReturnAttribute : Attribute { }
```
The public visibility is generating conflicts like 
`Warning	CS0436	The type 'DoesNotReturnAttribute' in 'C:\dev\Jwt\src\JsonWebToken\DoesNotReturnAttribute.cs' conflicts with the imported type 'DoesNotReturnAttribute' in 'gfoidl.Base64, Version=1.1.765.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'C:\dev\Jwt\src\JsonWebToken\DoesNotReturnAttribute.cs'.`

Comment (gfoidl) [reply source/gfoidl.Base64/ThrowHelper.cs]: Perfect, thanks 👍

**Comment by ycrumeyrolle** [comment]
Looks good

**Comment by gfoidl** [comment]
Thanks having a look. 
I'll have another look too, then I'll merge it (later today).

**Pull request merged**